src/DssSpell.sol

```diff
@@ -266,6 +313,56 @@ contract DssSpellAction is DssAction {
         // NOTE: ignore in goerli
 
 
+        // --------- Onboard GNO to Spark ---------
+        // Poll: https://vote.makerdao.com/polling/QmXdGdxS#poll-detail
+        // Forum: https://forum.makerdao.com/t/onboarding-of-gno-to-spark/20831
+        // List of addresses: https://github.com/marsfoundation/sparklend/blob/master/script/output/5/spark-latest.json
+        {
+            // Whitelist the GNO Fig adapter
+            MedianAbstract(GNO_MEDIANIZER).kiss(SPARK_GNO_ORACLE);
+
+            // Set DAI as a borrowable asset in isolation mode
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setBorrowableInIsolation(MCD_DAI, true);
+
+            // Add GNO
+            address token = DssExecLib.getChangelogAddress("GNO");
+            PoolConfiguratorLike.InitReserveInput[] memory input = new PoolConfiguratorLike.InitReserveInput[](1);
+            input[0] = PoolConfiguratorLike.InitReserveInput({
+                aTokenImpl: SPARK_ATOKEN_IMPL,
+                stableDebtTokenImpl: SPARK_STABLE_DEBT_TOKEN_IMPL,
+                variableDebtTokenImpl: SPARK_VARIABLE_DEBT_TOKEN_IMPL,
+                underlyingAssetDecimals: GemAbstract(token).decimals(),
+                interestRateStrategyAddress: SPARK_INTEREST_RATE_STRATEGY,      // Dummy strategy - compare to other borrow-disabled asset like sDAI
+                underlyingAsset: token,
+                treasury: SPARK_TREASURY,
+                incentivesController: address(0),
+                aTokenName: "Spark GNO",
+                aTokenSymbol: "spGNO",
+                variableDebtTokenName: "Spark Variable Debt GNO",
+                variableDebtTokenSymbol: "variableDebtGNO",
+                stableDebtTokenName: "Spark Stable Debt GNO",
+                stableDebtTokenSymbol: "stableDebtGNO",
+                params: ""
+            });
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).initReserves(input);
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).configureReserveAsCollateral({
+                asset: token, 
+                ltv: 2000,
+                liquidationThreshold: 2500,
+                liquidationBonus: 11000
+            });
+            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setDebtCeiling(token, 5 * MILLION * DEBT_CEILING_UNITS);
+
+            address[] memory tokens = new address[](1);
+            tokens[0] = token;
+            address[] memory oracles = new address[](1);
+            oracles[0] = SPARK_GNO_ORACLE;
+            AaveOracleLike(SPARK_AAVE_ORACLE).setAssetSources(
+                tokens,
+                oracles
+            );
+        }
```

Comment:
I've noticed there to be a common practice to comment the actions with the points from the checklist.

This would be especially helpful now with this new type of spell actions. E.g. reviewing things would be way faster.  Instead of going through the whole contract here and the contracts on etherscan to make sure the requirement is fulfilled one could just follow the call of the specific function to make sure that the requirement is fulfilled.

So please add the comments (e.g. copying from the exec sheet) next to the lines that fulfill the requirement